app/src/main/java/com/owncloud/android/ui/activity/BaseActivity.java

```diff
@@ -109,7 +109,10 @@ protected void onNewIntent(Intent intent) {
     protected void onRestart() {
         Log_OC.v(TAG, "onRestart() start");
         super.onRestart();
-        mixinRegistry.onRestart();
+        //Fix of NMC-2303 and NMC-2441
```

Comment:
Remove the code once the upstreamed PR is merge by NC: https://github.com/nextcloud/android/pull/11818